src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc

```diff
@@ -80,7 +80,7 @@ Real64 CoolingAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, b
             } else if (this->zoneEqSizing(this->curZoneEqNum).DesignSizeFromParent) {
                 this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).AirVolFlow;
             } else {
-                auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingAirflowSizing));
+                auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(static_cast<int>(AutoSizingType::CoolingAirFlowSizing)));
```

Comment:
The Zone Equipment Sizing structure has a member variable called `SizingMethod`.  This member is an Array1D of integers, allocated to the sizing the of sizing types list.  Each item in this array represents a more specific sizing attribute within that sizing type.  By the time I'm done this could look pretty different, but for right now the first thing I needed to do was get all these parts of the code away from the duplicated DataHVACGlobals int usage.

In the next commit I'll be making this a std::array...